main.py

```diff
@@ -18,4 +19,27 @@ def users():
     ]
   })
 
+@app.route("/api/events")
+def events():
+  spreadsheetId = '1MQ5wMn7Q3AKmXnpxcXyh5yPAPwhUCkNEvQAAczZurEU'
+  fields = 'sheets(properties.title,data.rowData.values.formattedValue)'
+  ranges = '*!A2:H20'
+  api_key = ''
+  # Can't specify range because it would only return first spreadsheet.
+  req = requests.get(f'https://sheets.googleapis.com/v4/spreadsheets/{spreadsheetId}?fields={fields}&key={api_key}')
+  res = req.json()
+  formatted_event_data = {}
+  for sheet in res['sheets']:
+    month = sheet['properties']['title']
+    # Event data from specified fields
+    events_data_raw = sheet['data'][0]['rowData'][1:]
+    monthly_formatted_events = []
+    for event_data_raw in events_data_raw:
+      formatted_event = []
+      for event_value in event_data_raw['values']:
+        formatted_event.extend(list(event_value.values()))
+      monthly_formatted_events.append(formatted_event)
+    formatted_event_data[month] = monthly_formatted_events
+  return jsonify(formatted_event_data)
```

Comment:
Please wrap the whole request in `Try...Expect` plus include some error handling.
For an example, the `API_KEY` is invalid or if the HTTP request didn't return an `ok` response.